can/listener.py

```diff
@@ -105,13 +105,14 @@ def on_message_received(self, msg: Message) -> None:
 
     def get_message(self, timeout: float = 0.5) -> Optional[Message]:
         """
-        Attempts to retrieve the latest message received by the instance. If no message is
-        available it blocks for given timeout or until a message is received, or else
-        returns None (whichever is shorter). This method does not block after
+        Attempts to retrieve the message that has been in the queue for the longest amount
+        of time (FIFO). If no message is available, it blocks for given timeout or until a
+        message is received, or else returns None (whichever is shorter). This method does
```

Comment:
I think it would be clearer to say "... until a message is recieved (whichever is shorter), or else returns None." otherwise it sounds like the it should return None because it is the shortest thing to do :smile: .